core/src/v2/models.rs

```diff
@@ -569,6 +586,10 @@ pub struct Operation<P, R> {
     pub consumes: Option<BTreeSet<MediaRange>>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub produces: Option<BTreeSet<MediaRange>>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    pub security: Vec<BTreeMap<String, Vec<String>>>,
+    #[serde(skip)]
+    pub security_definitions: BTreeMap<String, SecurityScheme>,
```

Comment:
I think the only reason this exists is  because one of the updating functions have this signature:
```rust
fn update_security(op: &mut DefaultOperationRaw) {}
```
I'll take care of cleaning this up later.

Reply:
And you had to do this because the operation function itself has the following signature:
```rust
fn operation() -> DefaultOperationRaw;
```

Reply:
Feel free to finish up with the implementation. I think it's better if I refactor everything and have a global context to take care of some stuff at the operation level.